spec/models/manageiq/providers/openstack/network_manager/cloud_network_spec.rb

```diff
@@ -45,16 +45,30 @@
     end
 
     context "#update_cloud_network" do
-      it 'catches errors from provider' do
+      it "updates the cloud network" do
+        options = {"name" => "new-name"}
+
+        expect(service).to receive(:update_network).with(cloud_network.ems_ref, options)
+        cloud_network.update_cloud_network(options)
+      end
+
+      it "catches errors from provider" do
         expect(service).to receive(:update_network).and_raise(bad_request)
-        expect { cloud_network.raw_update_cloud_network({}) }.to raise_error(MiqException::MiqNetworkUpdateError)
```

Comment:
We also called `raw_update_cloud_network` directly without going through the proper interface method